vendor/k8s.io/utils/mount/mount_windows.go

```diff
@@ -230,17 +229,17 @@ func (mounter *SafeFormatAndMount) formatAndMountSensitive(source string, target
 
 	// format disk if it is unformatted(raw)
 	cmd := fmt.Sprintf("Get-Disk -Number %s | Where partitionstyle -eq 'raw' | Initialize-Disk -PartitionStyle MBR -PassThru"+
-		" | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem %s -Confirm:$false", source, fstype)
+		" | New-Partition -UseMaximumSize | Format-Volume -FileSystem %s -Confirm:$false", source, fstype)
 	if output, err := mounter.Exec.Command("powershell", "/c", cmd).CombinedOutput(); err != nil {
 		return fmt.Errorf("diskMount: format disk failed, error: %v, output: %q", err, string(output))
 	}
 	klog.V(4).Infof("diskMount: Disk successfully formatted, disk: %q, fstype: %q", source, fstype)
 
-	driveLetter, err := getDriveLetterByDiskNumber(source, mounter.Exec)
+	volumeIds, err := listVolumesOnDisk(source)
 	if err != nil {
 		return err
 	}
-	driverPath := driveLetter + ":"
```

Comment:
this will cause Unmount failure, since original `Unmount` will remove `driveLetter + ":"` simply, now the target is volumeId
```
// Unmount unmounts the target.
func (mounter *Mounter) Unmount(target string) error {
	klog.V(4).Infof("azureMount: Unmount target (%q)", target)
	target = NormalizeWindowsPath(target)
	if output, err := exec.Command("cmd", "/c", "rmdir", target).CombinedOutput(); err != nil {
		klog.Errorf("rmdir failed: %v, output: %q", err, string(output))
		return err
	}
	return nil
}
```

Reply:
so with volumeId, unmount will fail? 
I think I manually tested it and it can pass. I will test it again.

Reply:
I am pretty sure the unmount would fail with this PR, I have reproed the unmount failure again in my env. `IsLikelyNotMountPoint` func does not work well with volumeID, here is the error:
```
azureDisk - TearTownAt:IsLikelyNotMountPoint check failed: CreateFile c:\\var\\lib\\kubelet\\pods\\84444347-81ef-48de-84d9-702f43295565\\volumes\\kubernetes.io~azure-disk\\pvc-2f4ae459-3cc0-419b-a1e8-5b0a29acfa30: The system cannot find the file specified."
```